core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.html

```diff
@@ -16,17 +16,17 @@
     <th class="dashboard-table-headings">
       <p> Question Content </p>
     </th>
-    <th class="dashboard-table-headings" ng-if="!$ctrl.getSkill()">
+    <th class="dashboard-table-headings" ng-if="$ctrl.getTopicId()">
       <p> Skill Description </p>
     </th>
   </tr>
-  <tr ng-repeat="question in $ctrl.questionSummaries"
+  <tr ng-repeat="questionSummary in $ctrl.questionSummaries"
       class="list-item protractor-test-question-list-item"
       style="height: 4em;"
-      ng-click="$ctrl.editQuestion(question.summary)">
+      ng-click="$ctrl.editQuestion(questionSummary.summary)">
     <td> <[$ctrl.getQuestionIndex($index)]> </td>
     <td> <[$ctrl.truncatedQuestionSummaries[$index]]> </td>
-    <td ng-if="!$ctrl.getSkill()">{{question.skill_description}}</td>
+    <td ng-if="$ctrl.getTopicId()"> <[$ctrl.getSkillDescription(questionSummary.skill_descriptions)]> </td>
   </tr>
 </table>
 <h4 ng-if="$ctrl.questionSummaries.length === 0" style="color: gray;">
```

Comment:
@sophiewu6 While you are waiting for a response from @aks681 @nithusha21 , it seems to me that all the suggestions stuff is from line 36 onwards (just judging by the names)? Though there seems to be a page navigation component in lines 67-73 that we might either want to factor out into a separate directive or move higher.

For now, would it work to keep lines 1-34 in the reusable "question list" directive and have the rest stay in the topic editor page?

Also @aks681 @nithusha21 PTAL ASAP, thanks!

Reply:
Thanks, but I didn't mean the code, I was talking about after starting my own local stack. I need that to manually test that I didn't break anything.

Reply:
Ah ok, thanks! I think @nithusha21 is chatting with you now, so hopefully that will help unblock you soon.

Reply:
As an update, I checked with @aks681 about this and he said he's not so familiar with the suggestions functionality; he thinks @nithusha21 is the person to ask.

@nithusha21 @sophiewu6 did you manage to resolve this issue?

Reply:
I am still trying out the way @nithusha21 told me. I ran into a bug while doing this. It also happens in the develop branch. I will open an issue regarding that.

Reply:
issue https://github.com/oppia/oppia/issues/7001

Reply:
Hi @seanlip @nithusha21 It seems to me that the suggestion feature is part of the question list. It's like there are two sections in the questions list - questions included and questions suggested. What do you think?

I think it might be a better idea to have a question suggested to a skill rather than a topic. It would make more sense to me since we always create a question targeting to a skill, not a topic. Even in the topic editor, we select the skill the question is linked to.

![image](https://user-images.githubusercontent.com/32754302/59990126-d801c980-95f6-11e9-967b-94243da47ad3.png)

Reply:
Thanks for investigating, @sophiewu6! Yes, I agree that questions should be suggested to skills. I thought that was what @DubeySandeep was working on in the contributor dashboard, in fact.

@DubeySandeep @nithusha21 should we remove the suggestions functionality from here and reinstate it again later when the contributor dashboard flows are done?

Reply:
I think that can be done. I can remove the question suggestions from the frontend, and we can add it back with the contributor dashboard. Some of the existing flows are broken as we haven't tested them in a while (and it wasn't released). Also, the long term goal is have this in the contributor dashboard, so I think removing it from here makes sense. I can do this in about 10 hours and hopefully this will unblock Shiqi?